src/pages/nft/collection/_collectionId/index.vue

```diff
@@ -249,9 +249,10 @@ export default {
         logPurchaseFlowEvent(this, 'add_to_cart', purchaseEventParams);
         logPurchaseFlowEvent(this, 'begin_checkout', purchaseEventParams);
         const customPriceInDecimal =
-          this.customPrice > -1
-            ? this.formatCustomPrice(this.customPrice, this.collectionPrice)
-            : undefined;
+          this.customPrice === 0
```

Comment:
Prefer checking falsy (`!this.customPrice`) or range comparison like `<= 0`

Reply:
[🎨 Update customPrice checking](https://github.com/likecoin/liker-land/pull/1554/commits/bba98bf23dff26fe297876231a72de50ddcb6be5)